src/WebJobs.Script.WebHost/Management/MeshServiceClient.cs

```diff
@@ -126,8 +130,8 @@ private async Task PublishActivities(IEnumerable<ContainerFunctionExecutionActiv
 
             var operation = new[]
             {
-                new KeyValuePair<string, string>(Operation, "add-fes"),
-                new KeyValuePair<string, string>("content", JsonConvert.SerializeObject(activities)),
+                new KeyValuePair<string, string>(Operation, AddFES),
+                new KeyValuePair<string, string>("content", Serialize(activities)),
```

Comment:
You could consider simplifying here by just using the static JsonConvert.Serialize(object, JsonSerializerSettings) overload, and using a statically defined set of settings.

Reply:
I tried that initially but it doesnt seem to work unless we use JsonSerializer.Create.

https://www.newtonsoft.com/json/help/html/M_Newtonsoft_Json_JsonSerializer_Create_1.htm

Also noticed similar pattern in https://github.com/Azure/azure-functions-host/blob/cbf9ad1b107c52531975f622d7f50f63ab31e592/src/WebJobs.Script/Timer/AzureStorageScheduleMonitor.cs#L44

Reply:
I agree with @mathewc -- I think you can make this a smaller change with https://www.newtonsoft.com/json/help/html/M_Newtonsoft_Json_JsonConvert_SerializeObject_5.htm.

```
private static readonly JsonSerializer _serializerSettings = new JsonSerializerSettings();
```

and then...

```
new KeyValuePair<string, string>("content", JsonConvert.SerializeObject(activities, _serializerSettings)),
```

will that work? I think that's typically the pattern we use.

Reply:
No. That doesn't seem to work and based on my searching around JsonSerializer.Create seems to be the only way to avoid using the global settings

From https://www.newtonsoft.com/json/help/html/M_Newtonsoft_Json_JsonSerializer_Create_1.htm
Creates a new JsonSerializer instance using the specified JsonSerializerSettings. The JsonSerializer **will not use** default settings from DefaultSettings.

Reply:
If you pass settings to JsonConvert.SerializeObject() as the second param, it'll use those instead of global:

From: https://www.newtonsoft.com/json/help/html/M_Newtonsoft_Json_JsonConvert_SerializeObject_5.htm
The JsonSerializerSettings used to serialize the object. If this is null, default serialization settings will be used.

Are you saying that this still uses the global settings?

Reply:
Yes. It still uses the global one.

Reply:
Sounds like we might have a different issue somewhere. I'm also concerned about positioning this as an option that would unblock the modification of global serialization settings, as there are other code paths that rely on that as well.

Reply:
Yeah there could be other places that needs similar fix as well as they are using global settings. This PR addresses one of those known issues that is resulting in CRIs. 

(Not trying to solve the problem of unblocking the modification of global serialization settings at all here :) This is much smaller in scope.)

Reply:
I'm concerned this would lead to a game of whack-a-mole. We'll address this just to have something down the road fail for the same reason and end up sprinkling similar code throughout the host. 

We should be crisp about this not being a supported pattern and work on a proper solution that would address the root of the problem.

Reply:
To be clear, I agree that this needs to be addressed, but we need to have a more generic approach we can leverage from the different code paths relying on serializer settings.

Reply:
I agree with that. But this could be a tactical fix? Do you have any alternative fixes we can try instead? The goal here is to reduce the CRIs resulting from this particular feature being broken.